### PR TITLE
Update prices_arbitrum_tokens.sql

### PR DESCRIPTION
**Pull request by 0xpibbles**
Hey I'd like to add a new Arbitrum token to the prices table - please let me know if I've missed anything (first time submitting a PR and I've tried following the guidelines)

### CODE REVIEW

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -103,5 +103,6 @@ FROM
     ('grain-granary','arbitrum','GRAIN',0x80bb30d62a16e1f2084deae84dc293531c3ac3a1,18),
     ('oath-oath','arbitrum','OATH',0xa1150db5105987cec5fd092273d1e3cbb22b378b,18),
     ('winr-winr-protocol','arbitrum','WINR',0xd77b108d4f6cefaa0cae9506a934e825becca46e,18),
-    ('ram-ramses-exchange','arbitrum','RAM',0xaaa6c1e32c55a7bfa8066a6fae9b42650f262418,18)
+    ('ram-ramses-exchange','arbitrum','RAM',0xaaa6c1e32c55a7bfa8066a6fae9b42650f262418,18),
+        ('emp-empyreal','arbitrum','EMP',0x772598E9e62155D7fDFe65FdF01EB5a53a8465BE,18)
```
[the remark this answers is not in the document]
Comment (Hosuke) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:107]: If the token you want to add is https://api.coinpaprika.com/v1/coins/emp-empyreal-exchange , it is currently not active for the price feed.

[the remark this answers is not in the document]
Comment (0xpibbles) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:107]: Hey thanks for the reply - it is indeed that token.

Is there anything I need to do on my end to activate it for the price feed?

[the remark this answers is not in the document]
Comment (Hosuke) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql:107]: We may need to wait for the coinpaprika to support its price feed, then we can add this token into `prices.usd` table.

[the remark this answers is not in the document]
Comment (Hosuke) [reply models/prices/arbitrum/prices_arbitrum_tokens.sql]: Maybe this token is no longer supported by the coinpaprika.
In this case, we may close this PR for now.

**Pull request closed without merging**